### PR TITLE
Patches

### PR DESCRIPTION
**Pull request by sirredbeard**
See commit descriptions.

### CODE REVIEW

File: linux_files/setup [hunk 1]
```diff
@@ -18,7 +18,7 @@ echo
 }
 
 function continueprompt {
-read -r -p "Would you like to continue with this setup script? [y/N] " response
+read -r -p "Would you like to continue with this setup script? [y/N] " response         # this says it defaults to N but when I hit enter to exit it continues the script anyways
 if [[ "$response" =~ ^[yY]$ ]] || [[ -z $response ]]
```
Comment (patrick330602) [inline linux_files/setup]: I think this is my mistake when I am editing... it suppose to be Y/n

Comment (sirredbeard) [reply linux_files/setup]: Gotcha. Thank you.

File: linux_files/setup [hunk 2]
```diff
@@ -325,19 +338,16 @@ function fzfinstall {
...
     ./install
-    cd $CURDIR
-    rm -r $TMPDIR
+    cleantmp
 fi
 }
 
 function explorerintegration {
-read -r -p "Would you like to enable Windows Explorer shell integration? [y/N] " response
+read -r -p "Would you like to enable Windows Explorer shell integration? [y/N] " response       # this says it defaults to N, which I think is correct, but hitting [enter] still attempts install
```
Comment (patrick330602) [inline linux_files/setup]: on line 360, `Install.reg` is accidentally copied as `Uninstall.reg`, I think this is the reason....

Comment (sirredbeard) [reply linux_files/setup]: Ah, I see. I will address with some user feedback.

**Comment by sirredbeard** [comment]
Thank you for your help reviewing these.

I will continue the script debugging tomorrow.

For now, sleep.

**Pull request merged**